tests/code_wars/test_vector_class.py

```diff
@@ -1,4 +1,4 @@
-import pytest
+import math, pytest
```

Comment:
`math` is a builtin import
'pytest` is a third party import

Format per [pep8](https://peps.python.org/pep-0008/#imports)